packages/client/src/object/convertWireToOsdkObjects.test.ts

```diff
@@ -78,6 +77,31 @@ describe("convertWireToOsdkObjects", () => {
     ]);
   });
 
+  it("stringifies properties on objects and interfaces correctly", async () => {
+    const { data: [employee] } = await client(Employee).fetchPage();
+    const { data: [employee2] } = await client(Employee).where({
+      $and: [{ employeeId: { $gt: 50030 } }, { employeeId: { $lt: 50032 } }],
+    }).fetchPage();
+
+    // Should not have $title
+    expect(JSON.stringify(employee)).toMatchInlineSnapshot(
```

Comment:
nit in the future if you `JSON.stringify(x, undefined, 2)` you get something much easier to read/review